build_posix/configure.ac.in

```diff
@@ -34,6 +34,15 @@ AC_PROG_CC(cc gcc)
 AC_PROG_CXX(c++ g++)
 AM_PROG_AS(as gas)
 
+# Set the libtool tag flag on PowerPC
+AS_CASE([$host_cpu],
```

Comment:
@daveh86 Is this only necessary when using clang? If so does it make sense to limit setting the flag to compiles using clang?